### PR TITLE
hardening/609_remove_persistOne_method

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #609 
* 100% unit test passed:
```
Results :
Tests run: 88, Failures: 0, Errors: 0, Skipped: 0
```
* Assignee @frbattid

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -14,3 +14,4 @@
 - [BUG] Add OrionPostgreSQLSink configuration template in cygnus_agent_conf.md (#769)
 - [BUG] Add missing sink types in agent.conf.templates (#773)
 - [BUG] Fix wrong data_model configured for OrionPostgreSQLSink in agent.conf.template (#774)
+- [HARDENING] Remove the persistOne method (#609)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: ... from all the sinks

Comment (pcoello25) [inline CHANGES_NEXT_RELEASE]: Fixed in 1fc5ad8

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionSink.java [hunk 2]
```diff
@@ -177,143 +177,12 @@ public void stop() {
     @Override
     public Status process() throws EventDeliveryException {
         // TBD: remove the special case of batchSize=1 when all the sinks have migrated to persistBatch
```
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionSink.java]: This comment must be removed

**Comment by frbattid** [comment]
In addition to the comments, as said in line 394 of OrionSink:

TBD: this class must be private once all the sinks migrate to persistsBatch

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionSink.java (hunk 2 above)
Comment (pcoello25) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionSink.java]: Fixed in 4974074

**Comment by pcoello25** [comment]
Done, fixed in 4974074

**Comment by frbattid** [comment]
LGTM

**Pull request merged**